markdown/usage/data_guidelines.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Data Management Guidelines
```

Comment:
How about the title: Resource Estimation and Data sharing.

It's more specific to the aim, and should serve to pull readers in looking to find that information.

Reply:
```suggestion
title: Resource Estimation and Data sharing
```